tests/unit/moduleapi/rdbloadsave.tcl

```diff
@@ -28,6 +20,7 @@ start_server {tags {"modules"}} {
     test "Module rdbloadsave test with pipelining" {
         r config set save ""
         r config set loading-process-events-interval-bytes 1024
+        r config set key-load-delay 50
```

Comment:
i noticed this test now takes 8 seconds, maybe we should tone it down?
the risk is that in some cases it'll be ineffective, right? but won't fail.
```
[ok]: Module rdbloadsave test with pipelining (7831 ms)
```

Reply:
Ok, used fewer keys, right now this test it takes 3-4 seconds on CI. I triggered a few daily's and it looks stable. 
I'm not sure if we can do it faster.

Reply:
can't we have just one key and a key-load-delay of 500ms?

Reply:
cut some more from tests, this above one takes ~500-600 ms now (also looks stable on daily builds).